docs/pages/eas/environment-variables.mdx

```diff
@@ -0,0 +1,193 @@
+When you create the file type environment variable `GOOGLE_SERVICES_JSON` with the value of the path to the **google-services.json** file in your project, EAS will upload the file to the EAS servers and make it available to your build job and workflow jobs running on EAS servers.
+During the job run, the file will be recreated in a random location on the filesystem and the path to the file will be assigned to the `GOOGLE_SERVICES_JSON` environment variable (`GOOGLE_SERVICES_JSON=/path/to/google-services.json`).
+You can then set `android.googleServicesFile` in your app config to the value of the `GOOGLE_SERVICES_JSON` environment variable to use this file when executing the build or workflow job.
+
+```js app.config.ts
+export default {
+  /* @hide ...*/ /* @end */
+  android: {
+    googleServicesFile: process.env.GOOGLE_SERVICES_JSON ?? '/local/path/to/google-services.json',
+    /* @hide ...*/ /* @end */
+  },
+};
+```
+
+## Managing environment variables and recommended workflow
+
+To learn how to create and manage environment variables in more details and set up the efficient workflow for using environment variables for your EAS Build jobs, EAS updates and local develpment see [Managing environment variables in EAS](/tutorial/eas/environment-variables) tutorial.
+
+## Differences between handling environment variables in EAS CLI and Expo CLI
+
+One of the differences between using environment variables in Expo framework and EAS is that, in opposition to Expo CLI, EAS CLI itself does not support loading **.env** files to set environment variables when resolving the expo config used to create your build jobs or updates.
+Instead, it's recommended to use the [EAS environment variables management system with EAS CLI commands](/tutorial/eas/environment-variables) to set environment variables for your build jobs and updates to avoid confusion, and ensure that exactly the same environment variables are used both for:
+
+- Local app config resolution - done by EAS CLI when preparing expo config needed for creating an update or triggering a build
+- Remote jobs - happening on EAS servers, which often don't have access to your local **.env** files that are gitignored.
+
+Using `eas update` can be the one exception to this rule, as by default, for backward compatibility reasons, it will use the **.env** files present in your project directory to set environment variables for the update job, the same way [Expo CLI](/guides/environment-variables) does (it executes the `npx expo export` command under the hood).
+To ensure that you can use the same environment variables in your updates as in your build jobs, you can use the `--environment` flag with `eas update` command to force it to use **only** the environment variables set on the EAS servers instead of the **.env** files present in your project directory (it will ignore environment variables from **.env**).
```

Comment:
```suggestion
To ensure that you can use the same environment variables in your updates as your build jobs, you can use the `--environment` flag with the `eas update` command to force it to use **only** the environment variables set on the EAS' servers instead of the **.env** files present in your project directory. It will ignore environment variables from **.env**.
```